Digizuite/HttpAbstraction/RestClient.cs

```diff
@@ -61,6 +61,16 @@ public async Task<RestResponse> SendAsync(RestRequest request, CancellationToken
             return new RestResponse(response.StatusCode, null, content, activityId);
         }
 
+        public async Task<Stream> StreamAsync(RestRequest request, CancellationToken cancellationToken)
+        {
+            var url = request.Uri.AbsoluteUri;
+            var httpRequest = new HttpRequestMessage(HttpMethod.Get, url);
+
+            var httpResponseMessage = await _client.SendAsync(httpRequest, HttpCompletionOption.ResponseHeadersRead,
+                cancellationToken);
+            return await httpResponseMessage.Content.ReadAsStreamAsync();
```

Comment:
Don't use the client directly like this, forward to the SendRequest method, otherwise you don't get bodies and headers and such passed along.